lib/model/Fixture.js

```diff
@@ -30,8 +31,9 @@ const Mode = require(path.join(__dirname, 'Mode.js'));
 
 
 module.exports = class Fixture {
-  constructor(jsonObject) {
+  constructor(jsonObject, man) {
     this.jsonObject = jsonObject; // calls the setter
+    this.manufacturer = new Manufacturer(man);
```

Comment:
What about adding a getter and a setter for manufacturer in order to be able to change the manufacturer like the jsonObject?